.github/workflows/pytest.yml

```diff
@@ -0,0 +1,51 @@
+# .github/workflows/app.yaml
+name: Pytest
+
+on: [push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.7"]
```

Comment:
Should we include the latest stable python version here, to be made aware of upcoming incompatibilities as early as possible?

Reply:
Good idea, done